almond/config.json

```
@@ -1,17 +1,16 @@
 {
-  "name": "Almond",
-  "version": "1.1.2",
-  "slug": "almond",
-  "description": "The home server version of Almond",
+  "name": "Almond - Edge",
+  "version": "2.0.0-1",
```

Version numbers of add-ons are decoupled with the software it wraps.

The `-1` addition is not something we use.

```suggestion
  "version": "2.0.0",
```

---

It's a mistake to decouple version numbers from upstream software, and one that I hope to avoid with this addon. You and us both need to know what version of Almond is associated with the specific add-on build, otherwise triaging bugs becomes a nightmare.
Even if you consider yourself a "distro" (bad, outdated model) you'll want to track the upstream version and the build number separately.

---

> It's a mistake to decouple version numbers from upstream software

It is not. please adjust.